app/presenters/account_relationships_presenter.rb

```diff
@@ -36,6 +38,37 @@ def initialize(account_ids, current_account_id, **options)
 
   private
 
+  def domain_blocking_map
+    target_domains = @accounts.pluck(:domain).compact.uniq
+    uncached_domains = []
+    blocks_by_domain = target_domains.index_with(false)
+
+    # Fetch from cache
+    unless target_domains.empty?
+      cache_keys = target_domains.map { |domain| "exclude_domains:#{@current_account_id}:#{domain}" }
```

Comment:
This key `"exclude_domains:#{@current_account_id}:#{domain}"` is used a few times here ... could this be either a) a private method in this file (`domain_cache_key(domain)` or something), or b) pass in the full current_account object instead of just the id, and add a method like `current_account.to_domain_exclusion_cache_key(domain)` or whatever. Private method might be cleaner option.